ci-operator/config/openshift/elasticsearch-operator/openshift-elasticsearch-operator-tech-preview.yaml

```diff
@@ -0,0 +1,89 @@
+base_images:
+  base:
+    name: "4.7"
+    namespace: ocp
+    tag: base
+  ocp_builder_rhel-8-golang-1.15-openshift-4.6:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.15-openshift-4.6
+  ocp_builder_rhel-8-golang-1.15-openshift-4.7:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.15-openshift-4.7
+  ocp_builder_rhel-8-golang-openshift-4.6:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-openshift-4.6
+  openshift_release_golang-1.13:
+    name: release
+    namespace: openshift
+    tag: golang-1.13
+  openshift_release_golang-1.14:
+    name: release
+    namespace: openshift
+    tag: golang-1.14
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.14
+images:
+- from: base
+  inputs:
+    ocp_builder_rhel-8-golang-1.15-openshift-4.7:
+      as:
+      - registry.svc.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.7
+  to: elasticsearch-operator
+- dockerfile_path: olm_deploy/operatorregistry/Dockerfile
+  from: base
+  to: elasticsearch-operator-registry
+- dockerfile_path: Dockerfile.src
+  from: src
+  to: elasticsearch-operator-src
+promotion:
+  name: tech-preview
+  namespace: logging
+resources:
+  '*':
+    requests:
+      cpu: 200m
+      memory: 200Mi
+tag_specification:
+  name: "4.7"
+  namespace: ocp
+tests:
+- artifact_dir: /tmp/artifacts
+  as: unit
+  commands: FAKE_HOME_DIR=/tmp/home LOG_DIR=/tmp/artifacts/eo-logs ARTIFACT_DIR=${ARTIFACT_DIR}/eo-artifacts make test-unit
+  container:
+    from: src
+- artifact_dir: /tmp/artifacts
+  as: e2e-operator
+  steps:
+    cluster_profile: aws
+    test:
+    - as: test
+      cli: latest
+      commands: FAKE_HOME_DIR=/tmp/home LOG_DIR=/tmp/artifacts/eo-logs ARTIFACT_DIR=${ARTIFACT_DIR}/eo-artifacts make test-e2e-olm
+      from: src
+      resources:
+        requests:
+          cpu: 100m
+    workflow: ipi-aws
+- as: cluster-logging-operator-e2e
+  steps:
+    cluster_profile: aws
+    test:
+    - as: test
+      cli: latest
+      commands: FAKE_HOME_DIR=/tmp/home LOG_DIR=/tmp/artifacts/logging-logs ARTIFACT_DIR=${ARTIFACT_DIR}/logging-artifacts make test-e2e-olm
+      from: stable:cluster-logging-operator-e2e
```

Comment:
Can we assume using `stable` here or do we need to use `tech-preview` to point to corresponding CLO tech-preview branch?

Reply:
The base images in the preceding stanza are referenceable from 'stable'